common/server-sent-events.conf

```diff
@@ -4,5 +4,4 @@ proxy_cache off;
 # make event source work with nginx
 # http://stackoverflow.com/questions/13672743/eventsource-server-sent-events-through-nginx
 proxy_set_header Connection '';
-proxy_http_version 1.1;
```

Comment:
Please have a look at http://serverfault.com/a/801629 It provides much more detailed description of what is needed and what is not and why.